### ISSUE TITLE
Outdated information in approved (2023) CRISPR treatment Casgevy

### ISSUE THREAD

**Issue by d0choa** [opentargets/issues#3631]
https://news.vrtx.com/news-releases/news-release-details/vertex-and-crispr-therapeutics-announce-authorization-first

I suggest reviewing the mechanism of action and indication max phase.

@ireneisdoomed, feel free to close this ticket if this is tracked on the ChEMBL side.

**Comment by ireneisdoomed** [comment]
The approved indications are displayed, but there is other data missing in the page.
I've opened a ticket here https://github.com/chembl/chembl_data/issues/926

**Comment by d0choa** [comment]
@ireneisdoomed are we expecting this in chembl 36?

**Comment by ireneisdoomed** [comment]
Yes, their `DRUG_INDICATION` table now reflects the approvals for [sickel cell anemia](https://platform.opentargets.org/disease/MONDO_0011382), [thalassemia](https://platform.opentargets.org/disease/EFO_1001996), and [beta thalassemia](https://platform.opentargets.org/disease/Orphanet_848)

**Comment by ireneisdoomed** [comment]
Complete https://dev.opentargets.xyz/drug/CHEMBL5095474
